dm/dm/ctl/common/config.go

```diff
@@ -215,3 +216,35 @@ func validateAddr(addr string) error {
 	}
 	return nil
 }
+
+// SplitArgsRespectQuote splits args by space, but won't split space inside single or double quotes.
+func SplitArgsRespectQuote(line string) []string {
```

Comment:
May need test in tiup?

Reply:
do you mean check tiup dmctl can pass this parameter?

Reply:
yes, test the Quote

Reply:
`SplitArgsRespectQuote` is only used in interactive mode of dmctl, not relevant to started by tiup.

I manually checked shell can put quoted string in one argument.